### PR TITLE
Fixed building placement on Tiberian Sun low bridges

### PR DESCRIPTION
**Pull request by Mailaender**
A followup of https://github.com/OpenRA/OpenRA/pull/11441 that fixes some oversights.

### CODE REVIEW

**Comment by pchote** [comment]
:+1: fix confirmed.

**Comment by pchote** [comment]
This is an easily reviewable PR. It would be good to get this merged.

File: OpenRA.Mods.TS/Traits/Buildings/LowBridge.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ namespace OpenRA.Mods.TS.Traits
 	[Desc("Bridge actor that can't be passed underneath.")]
 	class LowBridgeInfo : ITraitInfo, Requires<BuildingInfo>
 	{
-		public readonly string TerrainType = "Road";
+		public readonly string TerrainType = "Bridge";
```
Comment (reaperrr) [inline OpenRA.Mods.TS/Traits/Buildings/LowBridge.cs]: You need to define actor speeds for this new terrain type, currently this makes low bridges unpassable by all actors.

Comment (Mailaender) [inline OpenRA.Mods.TS/Traits/Buildings/LowBridge.cs]: Fixed.

**Pull request merged**